packages/vue/src/components/molecules/SfGallery/SfGallery.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div class="sf-gallery">
     <div class="sf-gallery__stage">
+      <slot name="options" />
```

Comment:
@martaradziszewska I can't find the reason why you add this slot, help me, please